tftrt/examples/object_detection/object_detection.py

```diff
@@ -539,88 +544,102 @@ def benchmark_model(frozen_graph,
                 NUM_DETECTIONS_NAME + ':0')
 
             # load batches from coco dataset
-            for image_idx in tqdm.tqdm(range(0, len(image_ids), batch_size)):
-                batch_image_ids = image_ids[image_idx:image_idx + batch_size]
-                batch_images = []
-                batch_coco_images = []
-
-                # read images from file
-                for image_id in batch_image_ids:
-                    coco_img = coco.imgs[image_id]
-                    batch_coco_images.append(coco_img)
-                    image_path = os.path.join(images_dir,
-                                              coco_img['file_name'])
-                    image = _read_image(image_path, image_shape)           
-                    batch_images.append(image)
-
-                # run once outside of timing to initialize
-                if image_idx == 0:
+            for image_idx in tqdm.tqdm(range(0, num_images, batch_size)):
+
+                if use_synthetic:
+                    if image_shape is not None:
+                        batch_images = np.random.randint(256, size=(batch_size, 600, 600, 3))
```

Comment:
Should I define standard image size (600, 600, 3) before use it?

Reply:
change the default in benchmark_model to [600, 600, 3]